src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,133 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private T[] arrayList;
+    int size = 0;
+    int emptyPlaces = 10;
```

Comment:
The fields `arrayList` and `emptyPlaces` should be declared as private to adhere to encapsulation. Moreover, the `emptyPlaces` variable is redundant as the size of the array can be managed with just the `size` variable and the length of the `arrayList`.